elastic/beats
# [Ingest Management] Include packages in local metadata

## What does this PR do?

This PR adds a `local_metadata.elastic.agent.packages` information with packages actually running by agent.
In case package input is filtered out by capabilities or dynamic expressions it wont be reported. 

How to test:
- build agent
- deploy cloud kibana 8.0.0-SNAPSHOT
- set configuration
- get agent id from agent detail page
- check metadata using dev tool `GET .fleet-agents/_doc/{agent_id}`

## Why is it important?

#24901

## Checklist
